src/elm/Page/Shop/Viewer.elm

```diff
@@ -448,11 +448,11 @@ viewCard ({ shared } as loggedIn) card model =
                             [ text (tr "account.my_wallet.your_current_balance" [ ( "balance", currBalance ) ]) ]
                         ]
                     ]
-                , div [ class " mt-6 md:mt-0" ]
+                , div [ class " mt-6 md:mt-0 w-full md:w-40 lg:w-40" ]
```

Comment:
`md:w-40` works for 768px and above, so we don't need `lg:w-40` here.

Also, it would be nice to remove that extra space at the beginning of the class string.

Reply:
Ok @amiskov sorry for the delay I'll make the changes right now.

Reply:
It looks like you're still in progress on this (maybe need to push changes to github?). Please, re-request review when it's ready.